### DISCUSSION TITLE
Buridan UI: Inspired by Shadcn UI Built for Python!

### DISCUSSION THREAD

**Discussion by LineIndent** [shadcn-ui/ui#5884] [Show and tell]
Hey everyone! 👋

I wanted to take a moment to share a UI component library I’ve been working on, inspired by the incredible work of the shadcn-ui team.

Shadcn-ui has been a massive source of inspiration for its clean, modern design and amazing functionality. With that in mind, I’ve created a similar component library tailored specifically for the Reflex framework, using Python. It’s been a fun and rewarding journey, and I really wanted to give a shout-out to the shadcn-ui community for sparking this idea!

The library aims to bring a seamless, intuitive UI experience to Python developers, leveraging the full power of Reflex. Thanks again to shadcn-ui and the team for being such an awesome source of inspiration! 🙌

Check it out:

    Site: https://native.buridan.dev/   
    Repo: https://github.com/LineIndent/native

Cheers! 👾

**Comment by arct0r** [comment]
Seems awesome man.

**Comment by amballa** [comment]
Thanks Ahmad. Your YouTube channel has been a source of immense help building with Reflex. Can't wait to check out this new library! Seems like current site is down though :(

**Reply by LineIndent** [reply]
Just updated the URL!

**Comment by hahelui** [comment]
Man This Is awesome.

**Comment by TheRaj71** [comment]
Amazing Work

**Comment by freecolortool** [comment]
Looks like a clean project! One thing I always pay attention to in UI systems is keeping icons visually consistent with the design theme. A [Recolor SVG ](https://freecolortool.com/svg-recolor.html)workflow makes it easy to adapt SVG icons to different color schemes without editing each file manually, which is especially useful when building reusable UI components.
